src/components/ChatApp/MessageComposer.react.js

```diff
@@ -12,13 +12,15 @@ import Modal from 'react-modal';
 import ReactFitText from 'react-fittext';
 import Close from 'material-ui/svg-icons/navigation/close';
 import TextareaAutosize from 'react-textarea-autosize';
-import './ChatApp.css'
+import './ChatApp.css';
+import YTSearch from 'youtube-api-search';
 injectTapEventPlugin();
 
 let ENTER_KEY_CODE = 13;
 let UP_KEY_CODE = 38;
 let DOWN_KEY_CODE = 40;
 var flag=1;
+const API_KEY = 'AIzaSyB9hQRq9nEns5OMBdSdCRx0o2Bo14IYPVM';
```

Comment:
API keys aren't usually exposed in the code.

Reply:
Usually, people use a config file to keep all their api keys in and don't commit that to the repo, they just commit a sample file for developers where developers can put in THEIR api keys. But I don't know if this is the right way to implement this but for some reason this works really well.